hugo-site/content/information/heritage.md

```diff
@@ -1,100 +1,18 @@
-
-Many of these differences from FreeBSD actually come from the TrueOS project, so this list will change over time as the TrueOS and FreeBSD projects continue to evolve.
-
-### OpenRC
-
-TrueOS provides the OpenRC system for managing services on the OS in place of the "legacy" RC system currently used on FreeBSD. This difference is most notable in a few key ways:
-
-* Faster bootup - OpenRC starts up services much faster and with better dependency handling than FreeBSD.
-* Better status reporting. The `rc-status` utility provides an instant look at all system services and their current status.
-* Service files are located in `[/usr/local]/etc/init.d` rather than `[/usr/local]/etc/rc.d`.
-* Some services have different names: **dhcpcd** instead of *dhclient*, **network** instead of *netif*.
-* Many OpenRC services are "multiplexed" for individual device/profile management.
-   * Example: The **network.wlan0** service controls the first wifi device, and should be started/stopped instead of the **network** service when re-configuring the Wi-Fi settings.
-* The *[service-name]_enable="YES"* entries on FreeBSD are no longer needed to enable services on bootup. Instead, OpenRC provides a method for grouping services together (called "runlevels"), and the `rc-update` utility is used to add/remove bootup registrations for services within these groups.
-   * Example: `rc-update add dbus default` adds the **dbus** service to the **default** runlevel and enables it to automatically start on bootup. 
-   * Example 2: `rc-update delete dbus default` removes the **dbus** service from the **default** runlevel and prevents it from starting during bootup.
-* The "service" utility is used identically to FreeBSD for interacting with services.
-   * Example: `service dbus stop` stops the **dbus** service when it is running.
-   * Example 2: `service network.wlan0 restart` restarts the networking associated with the first wireless device.
-
-### Base Packages 
-
-TrueOS uses the package system for installing the OS itself in addition to the pre-compiled "ports". This functionality is called "base packages" and allows a single package repository to contain both the OS and all the available applications in a single, unified collection of files. This functionality provides these features:
-
-* The OS and packages are always in sync. A single system update can be used for the entire end-to-end update of the system, even when OS changes like a new kernel are provided during a major release cycle.
-* System updates are **fast** and clean. All updates are performed within a new Boot Environment, ensuring the live system is never touched while in use. Reboot to finish the update procedure and boot into the newly-updated system.
+In 2019, Project Trident announced it would be leaving TrueOS/FreeBSD and rebasing the project upon [Void Linux](http://voidlinux.org) in order to better serve their users and provide essential functionality for a desktop system.
```

Comment:
```suggestion
In 2019, Project Trident announced it would be leaving TrueOS/FreeBSD and rebasing the project on [Void Linux](http://voidlinux.org) to better serve their users and provide essential functionality for a desktop system.
```